src/main/resources/org/jenkinsci/plugins/github/config/GitHubServerConfig/config.groovy

```diff
@@ -6,21 +6,26 @@ def f = namespace(lib.FormTagLib);
 def c = namespace(lib.CredentialsTagLib)
 
 
-f.entry(title: _("Manage hooks"), field: "manageHooks") {
-    f.checkbox(default: true)
+f.entry(title: _("API URL"), field: "apiUrl") {
+    f.textbox(default: GitHubServerConfig.GITHUB_URL)
 }
 
 f.entry(title: _("Credentials"), field: "credentialsId") {
     c.select()
 }
 
-f.optionalBlock(title: _("Custom GitHub API URL"),
-        inline: true,
-        field: "customApiUrl",
-        checked: instance?.customApiUrl) {
-    f.entry(title: _("GitHub API URL"), field: "apiUrl") {
```

Comment:
But global configuration is for admins - technical users that should keep eye on configuration. We should expect that they are not silly users :)